ext/opcache/jit/zend_jit_gdb.c

```diff
@@ -317,8 +318,12 @@ static void zend_gdbjit_ehframe(zend_gdbjit_ctx *ctx)
 		DSV(-(int32_t)sizeof(uintptr_t));              /* Data alignment factor. */
 		DB(DW_REG_RA);                                 /* Return address register. */
 		DB(1); DB(DW_EH_PE_textrel|DW_EH_PE_udata4);   /* Augmentation data. */
+#if defined(__x86_64__) || defined(i386)
```

Comment:
Do you think we should use `ZEND_JIT_TARGET_X86` and `ZEND_JIT_TARGET_ARM64` as well in this file?

Reply:
I think - yes, but that macros don't make distinct between x86 and x86_64 yet.